discord_simde/members.tf

```
@@ -517,3 +517,23 @@ module "louis_levasseur" {
     (discord_role.comet.id)    = true  #comet A22
   }
 }
+
+module "lucas_doublet" {
+  source    = "./member"
+  api_token = var.api_token
+  server_id = discord_server.server.id
+  user_id   = "231818542377467904"
+  roles     = {
+    (discord_role.curieux.id) = true
+  }
+}
+
+module "alexandre_eberhardt" {    # faire un nouveau channel woolly ?
```

Commentaire sur la PR ok, mais pas dans le fichier voyons ^^
```suggestion
module "alexandre_eberhardt" {
```